language/go/fileinfo.go

```diff
@@ -81,9 +81,9 @@ type fileInfo struct {
 	// a line after a "+build" prefix.
 	tags []tagLine
 
-	// copts and clinkopts contain flags that are part of CFLAGS, CPPFLAGS,
-	// CXXFLAGS, and LDFLAGS directives in cgo comments.
-	copts, clinkopts []taggedOpts
+	// copts, cppopts, cxxopts and clinkopts contain flags that are part
+	// of CFLAGS, CPPFLAGS, CXXFLAGS, and LDFLAGS directives in cgo comments.
+	copts, cppopts, cxxopts, clinkopts []taggedOpts
```

Comment:
Nit: let's put `cppopts` before `copts` and `cxxopts`. This applies everywhere these identifiers show up together.

This is a logical ordering: `cppopts` are options for the preprocessor; `copts` and `cxxopts` are for the C and C++ compilers; `clinkopts` are for the linker.